core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -238,7 +242,11 @@ export class ContributionsAndReview
           this.SUGGESTION_LABELS[suggestion.status].text),
         labelColor: this.SUGGESTION_LABELS[suggestion.status].color,
         actionButtonTitle: (
-          this.activeTabType === this.TAB_TYPE_REVIEWS ? 'Review' : 'View')
+          this.activeTabType === this.TAB_TYPE_REVIEWS ? 'Review' : 'View'),
+        translationWordCount: (
+          this.isReviewTranslationsTab() && this.activeExplorationId) ? (
+            this.computeTranslationWordCount(
+              suggestion.exploration_content_html)) : undefined
```

Comment:
`exploration_content_html` is actually the content of the exploration (which may have changed after the translation was submitted). I think you want to get the content of the suggestion's change object.

Reply:
Done